src/frontends/onnx/frontend/src/op/reduce.cpp

```diff
@@ -208,6 +207,14 @@ namespace set_18 {
 ov::OutputVector reduce_max(const ov::frontend::onnx::Node& node) {
     return {make_ov_reduction_op<v1::ReduceMax>(node, node.get_ov_inputs().at(0), supported_types_v3, false)};
 }
+ov::OutputVector reduce_log_sum(const ov::frontend::onnx::Node& node) {
+    const ov::Output<ov::Node> sum_node =
+        make_ov_reduction_op<v1::ReduceSum>(node, node.get_ov_inputs().at(0), supported_types_v2, false);
+    return {std::make_shared<v0::Log>(sum_node)};
+}
+ov::OutputVector reduce_max(const ov::frontend::onnx::Node& node) {
+    return {make_ov_reduction_op<v1::ReduceMax>(node, node.get_ov_inputs().at(0), supported_types_v3, false)};
+}
```

Comment:
Looks like merge artifact, `reduce_max` is already defined above, so it currently appears twice within the `set_18` namespace:
https://github.com/openvinotoolkit/openvino/blob/484db1db51c9c4c1d6d7bafe7c57644a0283cc2e/src/frontends/onnx/frontend/src/op/reduce.cpp#L207-L209
Please remove the reduce_max duplication:
```suggestion
```

Reply:
removed thank you